app/src/androidTest/java/com/omelan/cofi/pages/settings/licenses/TimerSettingsTest.kt

```diff
@@ -0,0 +1,79 @@
+package com.omelan.cofi.pages.settings.licenses
+
+import android.content.Context
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createComposeRule
+import com.omelan.cofi.LocalPiPState
+import com.omelan.cofi.pages.settings.TimerSettings
+import com.omelan.cofi.ui.CofiTheme
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class TimerSettingsTest {
+    @get:Rule
+    val composeTestRule = createComposeRule()
+    private lateinit var context: Context
+
+    private fun renderTimerSettings() {
+        composeTestRule.setContent {
+            context = LocalContext.current
+            CofiTheme {
+                CompositionLocalProvider(
+                    LocalPiPState provides false,
+                ) {
+                    TimerSettings(goBack = { })
+                }
+            }
+        }
+    }
+
+    @Test
+    fun showTimerSettings() {
+        composeTestRule.onNodeWithTag("settings_timer_list_item_pip").assertIsDisplayed()
```

Comment:
I think you've missed something here
```suggestion
    fun showTimerSettings() {
        renderTimerSettings()
        composeTestRule.onNodeWithTag("settings_timer_list_item_pip").assertIsDisplayed()
```